packages/twenty-front/src/modules/ui/field/input/components/TextInput.tsx

```diff
@@ -37,19 +46,20 @@ export const TextInput = ({
   onClickOutside,
   onChange,
   copyButton = true,
+  shouldTrim = true,
```

Comment:
I don't understand why we need this parameter. Why should we not trim sometimes?

Reply:
The `<TextInput />` component has always been used where we don't want spaces to be filled in the input, like for the Number field. Now that we use it for a regular text field, it was impossible to add a space character.

Reply:
Still don't get it. <TextInput is used in FieldName or Name Singular, which allows adding spaces